### PR TITLE
Split test-packages

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -88,24 +88,41 @@ DEFAULT_BUILD_RULES += $(WMTSCAPABILITIES_FILE)
...
 endif
+ifeq ($(NGEO), TRUE)
+TEST_PACKAGES += ngeo
 endif
-
-ifeq ($(MOBILE), TRUE)
-DEFAULT_BUILD_RULES += test-packages-mobile
+else
+TEST_PACKAGES += docker
+ifeq ($(PRINT_VERSION), 2)
```
Comment (pvalsecc) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: I don't understand what this if and the next one is doing in the docker area.

File: c2cgeoportal/scaffolds/update/CONST_packages.yaml [hunk 2]
```diff
@@ -53,6 +31,45 @@ docker:
...
+        cmd: /usr/share/tomcat*/bin/version.sh | grep 'Server number' | awk '{{print $3}}'
+        version: 6.0
+    unzip:
+        cmd: unzip -v | head -n1 | awk '{{print $2}}'
+        version: 6.0
+    zip:
+        cmd: zip -v | head -n2 | tail -n1 | awk '{{print $4}}'
+        version: 3.0
+
+docker-print:
```
Comment (pvalsecc) [inline c2cgeoportal/scaffolds/update/CONST_packages.yaml]: You don't need java, zip, unzip to do print with docker. Docker is all you need.

**Comment by pvalsecc** [comment]
Ship it

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**